client/pages/dashboard/index.js

```diff
@@ -61,10 +61,10 @@ const Dashboard = ({ currentUser, orders }) => {
       setGender(currentUser.gender);
       setAge(currentUser.age);
       setBio(currentUser.bio);
-      setAddress(currentUser.shippingAddress.address);
-      setCity(currentUser.shippingAddress.city);
-      setPostalCode(currentUser.shippingAddress.postalCode);
-      setCountry(currentUser.shippingAddress.country);
```

Comment:
It causes an error when a new user without shipping address make a route to this page